ingestion_server/ingestion_server/elasticsearch_models.py

```diff
@@ -197,19 +201,11 @@ def get_authority_boost(meta_data, source):
         return authority_boost
 
     @staticmethod
-    def get_authority_penalty(meta_data, source):
-        authority_penalty = None
-        if meta_data and 'authority_penalty' in meta_data:
-            try:
-                authority_penalty = float(meta_data['authority_penalty'])
-                authority_penalty = _constrain_between(
-                    authority_penalty, low=0, high=100
-                )
-            except (ValueError, TypeError):
-                pass
-        else:
-            authority_penalty = get_authority_penalty(source)
-        return authority_penalty
+    def get_popularity(raw):
+        if not raw:
+            return None
+        popularity = raw * 100
+        return _constrain_between(popularity, low=1, high=100)
```

Comment:
Why 1 instead of 0 as the minimum here?  This will modify a large number of popularity values for Wikimedia Commons in a somewhat unnatural way, and will make many Flickr images equivalent in popularity that wouldn't otherwise be.  I don't have precise numbers at hand, but remember that the Pareto distribution applies to these types of numbers, and so a huge number of images likely have 'raw popularity' between 0 and 0.01 in the Flickr data set.  In the current form, this step will therefore modify the popularity value for a large number of results in a way that affects their relative ranking against each other.

Reply:
As I think about it, the better conversion would be: `raw * 99 + 1`.  This will still map the interval `[0, 1) -> [1, 100)`, but will do so in a way that preserves relative rankings.  Then the `_constrain_between` call will just be a safety check.

Reply:
Elasticsearch `rank_feature` fields need to be positive and non-zero. I see your point about making irrelevant flickr images boosted; I intended to counterweight that with authority but that isn't making the cut.

We'll use `normalized_popularity=None` for images where the popularity is zero, indicating no boost should be applied.